docker-compose.yaml

```diff
@@ -3,6 +3,7 @@ version: '3.4'
 volumes:
   postgres_data: {}
   redis_data: {}
+  pulp_data: {}
```

Comment:
@cutwater 
For a deploy using pulp-api/pulp-content-app/pulp-worker in separate containers, is it okay for MEDIA_ROOT to point to the same persistent volume?

The docs don't really say anything about that, so I pinged the #pulp_ansible folks. 

Assuming that is ok, this looks good to me.

Reply:
@alikins Due to persistent database we don't want artifacts to be wiped out when container is recreated to avoid inconsistency between database and filesystem.